src/equations/acoustic_perturbation_2d.jl

```diff
@@ -216,7 +218,7 @@ function boundary_condition_wall(u_inner, orientation, direction, x, t,
         flux = surface_flux_function(u_boundary, u_inner, orientation, equations)
     end
 
-    return flux
+    return flux # TODO: Check surface_flux_function
```

Comment:
Where can I find all the surface flux functions?

Reply:
For the equations from this file, we typically just use `flux_lax_friedrichs`